### DISCUSSION TITLE
How to get access localStorage Nuxt.config.ts version 3?

### DISCUSSION THREAD

**Discussion by Hari111142** [nuxt/nuxt#16308] [Questions]
### Describe the feature

Hi Team, 

I tried the below code it's working fine but my concern is that I need to store the data or value in the session storage and to use data globally

async function getToken() {
const response = await axios.get(
"api/url",
);
const data = await response;
return data;
}
(async () => {

return (await getToken()).data;

})()

let resToken = getToken();
console.log(resToken) // Promise { }

resToken.then(function(result) {
console.log(result.data); // "Some User token"
})

Thanks in Advance

### Additional information

- [X] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by danielroe** [comment]
I'm not really sure what you are asking.
